tests/stratified_sampler_test.py

```diff
@@ -102,3 +97,18 @@ def test_with_dataloader(group_ids: list[int], sampler: BaseSampler) -> None:
         iters += 1
         if iters == 2:
             break
+
+
+@pytest.mark.parametrize("drop_last", [True, False])
+def test_sized(group_ids: list[int], drop_last: bool) -> None:
+    sampler = StratifiedBatchSampler(
+        group_ids=group_ids,
+        num_samples_per_group=225,
+        multipliers=None,
+        shuffle=False,
+        sized=True,
+        drop_last=drop_last,
+    )
+    batches = [batch for batch in sampler]
+    assert len(batches) == sampler.epoch_length
```

Comment:
should you also test `len(sampler)`?

Reply:
Yes, the only reason I didn't is because pyright complains about sampler not being Sized on the grounds that it can return a float. I've replaced `sampler.epoch_length` with `len(sampler)` and added a `type: ignore`.